### PR TITLE
Add basic model Structure required by the FOP management app

### PR DESCRIPTION
**Pull request by weizhang05**

### CODE REVIEW

**Comment by weizhang05** [comment]
@tantantan277 request for review

File: src/main/java/seedu/address/model/privilege/Privilege.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package seedu.address.model.privilege;
+
+
+/**
+ * Determines the level of privilege of the student based on the privilege assigned.
+ * Smaller number means more privilege.
+ * <p>
```
Comment (tantantan277) [inline src/main/java/seedu/address/model/privilege/Privilege.java]: Is this p tag necessary?

File: src/main/java/seedu/address/model/role/Participant.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.privilege.Privilege;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Participant for FOP.
+ * <p>
```
Comment (tantantan277) [inline src/main/java/seedu/address/model/role/Participant.java]: Is the p tag necessary?

**Review by tantantan277 — COMMENTED** [review]
Beside the <p\> tag in two of the code comments, everything is fine

File: src/main/java/seedu/address/model/role/Participant.java (hunk 2 above)
Comment (weizhang05) [reply src/main/java/seedu/address/model/role/Participant.java]: hmm, for both it wasn't there initially but somehow was added probably by the system. but thanks for the notice! xoxo

**Pull request merged**